logprep/connector/http/input.py

```diff
@@ -364,12 +364,14 @@ def __init__(self, name: str, configuration: "HttpConnector.Config", logger: Log
         internal_uvicorn_config = {
             "lifespan": "off",
             "loop": "asyncio",
-            "timeout_graceful_shutdown": 0,
+            "timeout_graceful_shutdown": 5,
         }
         self._config.uvicorn_config.update(internal_uvicorn_config)
         self.port = self._config.uvicorn_config["port"]
         self.host = self._config.uvicorn_config["host"]
         self.target = f"http://{self.host}:{self.port}"
+        if self.messages is None:  # this is only for testing. Is set by pipeline_manager
+            self.messages = mp.Queue(maxsize=self._config.message_backlog_size)
```

Comment:
If we only need this in testing, can't we also set the attribute in the corresponding test class then? Besides, I just executed the test without these lines and all the test run just fine.

Reply:
in my setup the tests hang if run without these lines. So the ci pipline tests deadlocks also.

Reply:
ah okay strange, but okay. Thanks for clearing that up for me.